DataProcessing/datax-host/src/main/scala/datax/fs/HadoopClient.scala

```diff
@@ -66,7 +66,7 @@ object HadoopClient {
   }
 
   /***
-    * get the name of storage account from a wasb-format path
+    * get the name of storage account from a wasb-format or abfss-format path
     * @param path a hdfs path
     * @return the storage account name if there is storage account name in the wasbs/wasb path, else null
     */
```

Comment:
Can we rename getWasbStorageAccount to something more generic since this no longer is only for wasb format? I wasn't sure if it was okay to rename this or if that will break something.

Reply:
should be ok as long as you rename all instances